enigma/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation 'org.ow2.asm:asm-tree:9.4'
     implementation 'org.ow2.asm:asm-util:9.4'
 
+    implementation 'org.quiltmc:quiltflower:1.10.0-20221207.002314-21'
```

Comment:
Needs updating once QuiltMC/quiltflower#226 is fixed, shouldn't be merged before that

Reply:
Fixed, the pr is now ready to merge